### PR TITLE
Feature/tanks graph

### PR DESCRIPTION
**Pull request by Andileh**
Fixing UI for tank page and Implementing back button from tankInformation page to home

### CODE REVIEW

File: waterlog/src/Components/TankGraph.js [hunk 1]
```diff
@@ -47,36 +48,37 @@ const TankGraph = (props) => {
...
-          fontColor:'rgb(236,239,241)'
+          labelString: 'percentage level',
+          fontColor:'#ECEFF1'
         },
         ticks: {
           fontColor:'rgb(236,239,241)',
+        },
         gridLines: {
           display: false,
           color:'rgb(236,239,241)'
```
Comment (CilliersN) [inline waterlog/src/Components/TankGraph.js]: above changes was made from rgb to hex. why is this still rgb? Also, why was is changed to hex?

**Review by CilliersN — CHANGES_REQUESTED** [review]

File: waterlog/src/Pages/TankInformation.js [hunk 2]
```diff
@@ -1,41 +1,52 @@
...
+	constructor(props) {
+		super(props);
+		this.state = {
+			mapExpanded: false,
+			id: this.props.match.params.id
+		};
+	}
+	handleMapExpand() {
+		this.setState({
+			mapExpanded: !this.state.mapExpanded,
```
Comment (CilliersN) [inline waterlog/src/Pages/TankInformation.js]: remove floating comma

File: waterlog/src/Stylesheets/_segmentLeak.scss [hunk 3]
```diff
@@ -1,6 +1,12 @@
 #map-toggle {
     width: 75px;
 }
+#map-toggle1 {
+    width: 75px;
+    position: relative;
+    top:-115px;
+
```
Comment (CilliersN) [inline waterlog/src/Stylesheets/_segmentLeak.scss]: remove this line

File: waterlog/src/Stylesheets/_tankInfo.scss [hunk 4]
```diff
@@ -16,6 +16,8 @@
   border: none;
   line-height: 16sp;
   box-shadow: black 1dp, 1dp, 4dp;
+  top:-125px;
```
Comment (CilliersN) [inline waterlog/src/Stylesheets/_tankInfo.scss]: -125 ?

File: waterlog/src/Stylesheets/_segmentLeak.scss [hunk 5]
```diff
@@ -79,6 +85,14 @@
     height: 400px;
     margin: 25px 0;
 }
+.segment-map2 {
+    .segment-map {
+        height: 200px;
```
Comment (CilliersN) [inline waterlog/src/Stylesheets/_segmentLeak.scss]: line 85 is 400 and this is 200? why such large px? on what was this tested?

File: waterlog/src/Stylesheets/_tankInfo.scss [hunk 6]
```diff
@@ -45,13 +47,19 @@
...
-  //top: -110px;
+  top: -110px;
   left: 8px;
   position: relative;
 }
+.backIcon{
+  width: 26px;
+  position: absolute;
+  left: 11px;
+  bottom: 560px;
```
Comment (CilliersN) [inline waterlog/src/Stylesheets/_tankInfo.scss]: same question here with regards to px.

File: waterlog/src/reducers/rootReducer.js [hunk 7]
```diff
@@ -54,9 +54,10 @@ const rootReducer = combineReducers({
   forecastMonthly:CostsMonthlyForecastReducer,
   dailytankgraph:TankGraphReducer,
   heatMap:HeatMapsReducer,
+  heatMap:HeatMapsReducer,
+  placeholder:ForecastPlaceholderReducer,
   numNotifs : NotificationsReducer,
   filteredAlerts: FilterReducer,
   dailytankgraph: TankGraphReducer,
-  placeholder: ForecastPlaceholderReducer
```
Comment (CilliersN) [inline waterlog/src/reducers/rootReducer.js]: if this line is removed, remove floating comma in the above line.

File: waterlog/src/Stylesheets/_tankInfo.scss [hunk 8]
```diff
@@ -45,13 +47,19 @@
...
 .Level {
-  //top: -95px;
+  top: -95px;
   left: 5px;
   position: relative;
   color: #93a4ae;
 }
 .Instruction {
-  //top: -110px;
+  top: -110px;
```
Comment (CilliersN) [inline waterlog/src/Stylesheets/_tankInfo.scss]: again. negative px?

**Review by CilliersN — CHANGES_REQUESTED** [review]

File: waterlog/src/Components/TankGraph.js (hunk 1 above)
Comment (Andileh) [reply waterlog/src/Components/TankGraph.js]: The reason is that they said we have to use one either rgb ox Hex ,,,,The majority in our file were in hex so I've changed that to  hex as well.

File: waterlog/src/Pages/TankInformation.js (hunk 2 above)
Comment (Andileh) [reply waterlog/src/Pages/TankInformation.js]: done

File: waterlog/src/Stylesheets/_segmentLeak.scss (hunk 3 above)
Comment (Andileh) [reply waterlog/src/Stylesheets/_segmentLeak.scss]: done

File: waterlog/src/Stylesheets/_segmentLeak.scss (hunk 5 above)
Comment (Andileh) [reply waterlog/src/Stylesheets/_segmentLeak.scss]: i remove the second one  with 200 its  was for testing purposes ,its not used

File: waterlog/src/reducers/rootReducer.js (hunk 7 above)
Comment (Andileh) [reply waterlog/src/reducers/rootReducer.js]: done

File: waterlog/src/Stylesheets/_tankInfo.scss (hunk 4 above)
Comment (Andileh) [reply waterlog/src/Stylesheets/_tankInfo.scss]: I changed  top -px to bottom +px

File: waterlog/src/reducers/rootReducer.js (hunk 7 above)
Comment (Andileh) [reply waterlog/src/reducers/rootReducer.js]: done

File: waterlog/src/Stylesheets/_tankInfo.scss (hunk 8 above)
Comment (Andileh) [reply waterlog/src/Stylesheets/_tankInfo.scss]: changed

File: waterlog/src/Stylesheets/_tankInfo.scss (hunk 6 above)
Comment (Andileh) [reply waterlog/src/Stylesheets/_tankInfo.scss]: changed

File: waterlog/src/Stylesheets/_tankInfo.scss (hunk 4 above)
Comment (Andileh) [reply waterlog/src/Stylesheets/_tankInfo.scss]: changed to bottom positive px

File: waterlog/src/Stylesheets/_segmentLeak.scss (hunk 3 above)
Comment (Andileh) [reply waterlog/src/Stylesheets/_segmentLeak.scss]: done

File: waterlog/src/Components/TankGraph.js (hunk 1 above)
Comment (Andileh) [reply waterlog/src/Components/TankGraph.js]: the reason is that the majority that is used is hex. and they said i must changed it to hex .So I've changed that rgb to hex..

File: waterlog/src/Components/FaultySensor.js [hunk 9]
```diff
@@ -82,6 +82,10 @@ class FaultySensor extends Component {
 
 		const sensorInfo = (
 			<div>
+	     <img className = "back-icon"
```
Comment (FredJEhlers) [inline waterlog/src/Components/FaultySensor.js]: Indentation needs to be fixed

File: waterlog/src/Components/SegmentLeak.js [hunk 10]
```diff
@@ -83,7 +84,11 @@ class SegmentLeak extends Component {
 		const resolved = parseInt(status) === 1 ? true : false;
 		const leakInfo = (
 			<div>
-				<div className="leak-info">
+      	<img className = "back-icon"
```
Comment (FredJEhlers) [inline waterlog/src/Components/SegmentLeak.js]: indentation again

File: waterlog/src/Components/TankGraph.js [hunk 11]
```diff
@@ -9,16 +9,18 @@ const TankGraph = (props) => {
   let dataY = mapDataTime(myData); 
   let data = {
     labels: labelX,
-    datasets: [ 
+    datasets: [
+
```
Comment (FredJEhlers) [inline waterlog/src/Components/TankGraph.js]: remove unnecessary white space

File: waterlog/src/Pages/TankInformation.js.BASE [hunk 12]
```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
```
Comment (FredJEhlers) [inline waterlog/src/Pages/TankInformation.js.BASE]: Delete this file

File: waterlog/src/Pages/TankInformation.js.LOCAL [hunk 13]
```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
```
Comment (FredJEhlers) [inline waterlog/src/Pages/TankInformation.js.LOCAL]: Delete file

File: waterlog/src/Pages/TankInformation.js.REMOTE [hunk 14]
```diff
@@ -0,0 +1,77 @@
+import React, { Component } from "react";
```
Comment (FredJEhlers) [inline waterlog/src/Pages/TankInformation.js.REMOTE]: Delete this file

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

**Review by CilliersN — APPROVED** [review]

File: waterlog/src/Pages/TankInformation.js.LOCAL (hunk 13 above)
Comment (Andileh) [reply waterlog/src/Pages/TankInformation.js.LOCAL]: done

File: waterlog/src/Pages/TankInformation.js.REMOTE (hunk 14 above)
Comment (Andileh) [reply waterlog/src/Pages/TankInformation.js.REMOTE]: done

File: waterlog/src/Components/TankGraph.js (hunk 11 above)
Comment (Andileh) [reply waterlog/src/Components/TankGraph.js]: done

File: waterlog/src/Components/SegmentLeak.js [hunk 15]
```diff
@@ -84,7 +84,7 @@ class SegmentLeak extends Component {
 		const resolved = parseInt(status) === 1 ? true : false;
 		const leakInfo = (
 			<div>
-      	<img className = "back-icon" 
+        <img className = "back-icon"
```
Comment (FredJEhlers) [inline waterlog/src/Components/SegmentLeak.js:87]: still not fixed

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

**Review by FredJEhlers — APPROVED** [review]

**Pull request merged**